config/api.yml

```diff
@@ -4107,6 +4114,14 @@
       - :name: read
         :identifier:
         - vm_show
+    :compliances_subcollection_actions:
+      :get:
+      - :name: read
+        :identifiers:
+        - :klass: Vm
+          :identifier: vm_show
+          :identifier: vm_check_compliance
+          :identifier: instance_check_compliance
```

Comment:
I don't think this is valid YAML...

Reply:
```ruby
s = <<~YAML
       - :klass: Vm
         :identifier: vm_show
         :identifier: vm_check_compliance
         :identifier: instance_check_compliance
YAML
YAML.load(s)
# => [{:klass=>"Vm", :identifier=>"instance_check_compliance"}]
```

Notice the keys are lost.

Reply:
LOL...this is actually what rubocop found.

Reply:
yeah, we don't really handle multiple identifiers all that well anyway as it turns out